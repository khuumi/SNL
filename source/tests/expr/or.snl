5.429 or 42