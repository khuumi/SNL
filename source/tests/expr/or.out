Binop (Float 5.429) Or (Int 42)